app/components/IDDialog.tsx

```diff
@@ -21,6 +22,31 @@ export default ({ issueCredsStatus }: IDDialogProps) => {
     navigate('/');
   };
 
+  const renderTitle = () => {
+    if (issueCredsStatus === 'success')
+      return 'Congratulations! Your Hooli ID card is activated.';
+
+    if (isPartner)
+      return 'Earn $20 and get an AI financial assistant with Kredita';
+
+    if (issueCredsStatus === 'new')
+      return 'Get your Hooli digital ID card. \n Free forever.';
+
+    return 'We encountered an error while issuing your Unum ID. Please try again later';
```

Comment:
We don't use the `Unum ID` name any more, so this could be confusing if someone sees it.

```suggestion
    return 'We encountered an error while issuing your Hooli ID Card. Please try again later';
```